### PR TITLE
Simplify grid height/width logic 

### PR DESCRIPTION
**Pull request by amanmahajan7**

### CODE REVIEW

File: src/style/core.ts [hunk 1]
```diff
@@ -71,11 +71,8 @@ const root = css`
   /* needed on Firefox */
   &::before {
     content: '';
-    position: absolute;
-    inset-block-start: 0;
-    inset-inline-start: 0;
-    block-size: var(--rdg-grid-block-size);
-    inline-size: var(--rdg-grid-inline-size);
+    grid-column: 1/-1;
```
Comment (amanmahajan7) [inline src/style/core.ts:74]: This works in FF

File: test/virtualization.test.ts [hunk 2]
```diff
@@ -37,11 +37,6 @@ function setupGrid(
   return setup({ columns, rows, summaryRows, rowHeight, enableVirtualization });
 }
 
-function assertHeightFill(height: number) {
```
Comment (amanmahajan7) [inline test/virtualization.test.ts:40]: I don't think we can/should test the height now

File: test/virtualization.test.ts [hunk 3]
```diff
@@ -144,7 +137,7 @@ test('virtualization is enabled', () => {
   assertCells(66, 18, 1, 18);
 
   // max left = row width - grid width
-  grid.scrollLeft = parseInt(grid.style.getPropertyValue('--rdg-grid-inline-size'), 10) - 1920;
+  grid.scrollLeft = 3600 - 1920;
```
Comment (amanmahajan7) [inline test/virtualization.test.ts:140]: Hard to hard code the value

File: test/virtualization.test.ts [hunk 4]
```diff
@@ -227,23 +213,14 @@ test('zero columns', () => {
...
-  assertHeightFill(1080 - rowHeight + 35);
   expect(getHeaderCells()).toHaveLength(18);
   expect(queryCells()).toHaveLength(0);
   expect(queryRows()).toHaveLength(0);
 });
 
-test('zero rows + 2 summary rows', () => {
-  setupGrid(true, 20, 0, 0, 2);
-
-  assertHeightFill(1080 - rowHeight * 3 + 35 + 70);
```
Comment (nstepien) [inline test/virtualization.test.ts:239]: I wonder if it's worth testing the values of `--rdg-summary-row-bottom`/`--rdg-summary-row-top`

**Review by nstepien — APPROVED** [review]

**Comment by amanmahajan7** [comment]
Tested and worked great

File: test/virtualization.test.ts (hunk 4 above)
Comment (amanmahajan7) [reply test/virtualization.test.ts:239]: I will check in the next PR

File: test/virtualization.test.ts (hunk 3 above)
Comment (amanmahajan7) [reply test/virtualization.test.ts:140]: May be. I can try in the next PR

**Pull request merged**